pkg/catalogsourceconfig/configuring.go

```diff
@@ -124,6 +130,27 @@ func (r *configuringReconciler) reconcileCatalogSource(csc *v1alpha1.CatalogSour
 	return nil
 }
 
+// ensurePackages makes sure that the csc's status.PackageRepositioryVersions field is updated
+// at the end of the configuring phase if successful. It iterates over the list of packages
+// and creates a new map of PackageName:Version for each package in the spec.
+func (r *configuringReconciler) ensurePackagesInStatus(csc *v1alpha1.CatalogSourceConfig) error {
```

Comment:
Given you are never returning an error, is a return `error` still needed?

Reply:
good point. I updated this as well as removing the error handling where it is called.